Dangerfile

```diff
@@ -6,6 +6,11 @@ has_app_changes = !git.modified_files.grep(/Sources/).empty?
 # Make it more obvious that a PR is a work in progress and shouldn't be merged yet
 warn("PR is classed as Work in Progress") if github.pr_title.include? "WIP"
 
+# Warn, asking to update Chinese translation of docs if file(s) in `docs` is/are modified
+if !git.modified_files.grep(/docs/).empty?
```

Comment:
This is a good start, but we should be more clever. For example, if someone _does_ update both docs, this warning is still triggered. What do you think about this instead?

```rb
en_docs_modified = !git.modified_files.grep(/docs\//).empty? # Necessary to exclude `docs_CN` from the grep.
cn_docs_modified = !git.modified_files.grep(/docs_CN/).empty?
if en_docs_modified ^ cn_docs_modified
  warn(...)
end
```

This uses the exclusive-or (xor) operator, which is true if only one or the other is true (but not both). This way, if someone: 

- updates both, the xor will be false
- if they update neither, the xor will be false
- updates either one but not the other, the xor will be true

I think that makes sense. What do you think?

Reply:
No, no, completely agreed. I kinda had the feeling this would be brought up right after I made the PR 🙊

I will update the PR now. I love the use of the XOR operator too! Couldn't be more cleaner!